addons/mail/models/res_config_settings.py

```diff
@@ -27,6 +27,11 @@ class ResConfigSettings(models.TransientModel):
         config_parameter='mail.restrict.template.rendering',
         help='Users will still be able to render templates.\n'
         'However only Mail Template Editors will be able to create new dynamic templates or modify existing ones.')
+    use_call_server = fields.Boolean(
```

Comment:
Don't forget to update mail/models/ir_ config_parameter, which holds an explanation of all parameters used in mail (quite a lot ^^ )